### PR TITLE
LP-2679 [Safari] Design of ORA Table is getting disturbed

### PR DESCRIPTION
**Pull request by annas-usman**
[Ticket](https://philanthropyu.atlassian.net/browse/LP-2679)

### CODE REVIEW

File: openassessment/xblock/static/js/src/adg/lib/table_builder.js [hunk 1]
```diff
@@ -290,6 +290,19 @@ $.fn.oraTableBuilder = function ($config) {
             $(table).find('td').each(function () {
                 $(this).empty();
                 $(this).append($(_this.controls[value]).clone(true, true));
+
+                if($.browser.safari) {
+                    const checkbox = $(this).find('input');
```
Comment (MunamMubashir-arbisoft) [inline openassessment/xblock/static/js/src/adg/lib/table_builder.js:295]: Is it okay to use ES6 way of variable declaration?

File: openassessment/xblock/static/js/src/adg/lib/table_builder.js [hunk 2]
```diff
@@ -358,6 +371,19 @@ $.fn.initORATableCheckbox = function (checkboxClass, disabled) {
...
+                    prependCaption
+                );
+
+                function prependCaption() {
+                    const table = _this.find('table');
+                    const caption = _this.find('caption');
+                    caption.detach();
+                    table.prepend(caption);
+                }
+            }
```
Comment (MunamMubashir-arbisoft) [inline openassessment/xblock/static/js/src/adg/lib/table_builder.js:386]: Is there a way we might resuse the same code in two places?

File: openassessment/xblock/static/js/src/adg/lib/table_builder.js (hunk 1 above)
Comment (ahmed-faraz46) [inline openassessment/xblock/static/js/src/adg/lib/table_builder.js:295]: Let's please be as specific as possible in the selector, if we want to select only check boxes then change the selector accordingly.

File: openassessment/xblock/static/js/src/adg/lib/table_builder.js (hunk 2 above)
Comment (ahmed-faraz46) [reply openassessment/xblock/static/js/src/adg/lib/table_builder.js:386]: +1

**Review by ahmed-faraz46 — CHANGES_REQUESTED** [review]

**Review by MunamMubashir-arbisoft — APPROVED** [review]
Good work, LGTM!

**Review by javeriaejaz-arbisoft — APPROVED** [review]
LGTM

**Review by ahmed-faraz46 — APPROVED** [review]

**Pull request merged**